yt_dlp/extractor/linkedin.py

```diff
@@ -104,12 +104,17 @@ def _real_extract(self, url):
         like_count = int_or_none(get_element_by_class('social-counts-reactions__social-counts-numRections', webpage))
         creator = strip_or_none(clean_html(get_element_by_class('comment__actor-name', webpage)))
 
-        sources = self._parse_json(extract_attributes(self._search_regex(r'(<video[^>]+>)', webpage, 'video'))['data-sources'], video_id)
+        video_json = extract_attributes(self._search_regex(r'(<video[^>]+>)', webpage, 'video'))
+        sources = self._parse_json(video_json['data-sources'], video_id)
```

Comment:
`video_json` is a misnomer imo
```suggestion
        video_attrs = extract_attributes(self._search_regex(r'(<video[^>]+>)', webpage, 'video'))
        sources = self._parse_json(video_attrs['data-sources'], video_id)
```